projects/batfish/src/main/java/org/batfish/representation/iptables/IptablesChain.java

```diff
@@ -19,7 +19,7 @@
 
   private String _name;
 
-  private ChainPolicy _policy;
+  private ChainPolicy _policy = ChainPolicy.ACCEPT;
 
   private List<IptablesRule> _rules;
 
```

Comment:
to test this, can you just delete the `ACCEPT` line from the existing testrig? Maybe just delete this line: https://github.com/batfish/batfish/blob/ea39b4ccf537a67df0f477cbc921d4b7d88d7af2/test_rigs/unit-tests/iptables/host1.iptables#L5